examples/node-app-mssql-ts/docker-compose.yml

```
@@ -2,15 +2,16 @@ version: '3.3'
 
 services:
   mssql:
-    container_name: 'example-mssql'
+    container_name: 'example-mssql-ts'
```

This is just the database?

```suggestion
    container_name: 'example-mssql'
```